cmd/controllers/app/options/options.go

```diff
@@ -39,7 +40,9 @@ type ServerOption struct {
 	PrintVersion         bool
 	// WorkerThreads is the number of threads syncing job operations
 	// concurrently. Larger number = faster job updating,but more CPU  load.
-	WorkerThreads uint32
+	WorkerThreads            uint32
+	EnablePodgroupController bool
```

Comment:
What's impact if it's false?

Reply:
controller not started, so it does not create a PG for the pod.

Reply:
can we remove that? if pod selector is empty, not podgroup will be create for any pod.

Reply:
emmm，the EnablePodgroupController  is a switch for pg_controller,
if no need，all pg controller do not work.